src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -567,6 +722,22 @@ private void assertNoFeeAssignedForLoan(UUID loan) {
     assertThat(getAccountsForLoan(loan), hasSize(0));
   }
 
+  private List<JsonObject> getOpenAccounts(List<JsonObject> accounts) {
+    return accounts.stream().filter(account -> isOpenAccount(account)).collect(Collectors.toList());
+  }
+
+  private Boolean isOpenAccount(JsonObject account) {
```

Comment:
I think account can be omitted from this method name, given the name of the parameter. If you would prefer to include it, I think it reads better in my head as `is (the) account open`. If you don't prefer either of these, feel free to resolve without changes.

```suggestion
  private Boolean isOpen(JsonObject account) {
```

```suggestion
  private Boolean isAccountOpen(JsonObject account) {
```